ManagementPacks/Community.DataOnDemand/Modules/WatchTcpConnectionsWA.mpx

```diff
@@ -0,0 +1,68 @@
+﻿<ManagementPackFragment SchemaVersion="2.0" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+
+  <TypeDefinitions>
+    <ModuleTypes>
+      <WriteActionModuleType ID="Community.DataOnDemand.WriteAction.WatchTcpConnections" Accessibility="Public" Batching="false">
+        <Configuration>
+          <xsd:element name="Interval" type="xsd:int" xmlns:xsd="http://www.w3.org/2001/XMLSchema" />
+          <xsd:element name="Format" type="xsd:string" xmlns:xsd="http://www.w3.org/2001/XMLSchema" />
+          <xsd:element name="TimeoutSeconds" type="xsd:int" xmlns:xsd="http://www.w3.org/2001/XMLSchema" />
+        </Configuration>
+        <OverrideableParameters>
+          <OverrideableParameter ID="Interval" Selector="$Config/Interval$" ParameterType="int" />
+          <OverrideableParameter ID="Format" Selector="$Config/Format$" ParameterType="string" />
+          <OverrideableParameter ID="TimeoutSeconds" Selector="$Config/TimeoutSeconds$" ParameterType="int" />
+        </OverrideableParameters>
+        <ModuleImplementation>
+          <Composite>
+            <MemberModules>
+              <WriteAction ID="WA" TypeID="Windows!Microsoft.Windows.PowerShellWriteAction">
+                <ScriptName>Watch-TcpConnections.ps1</ScriptName>
+                <ScriptBody>$IncludeFileContent/Scripts/Watch-TcpConnections.ps1$</ScriptBody>
+                <Parameters>
+                  <Parameter>
+                    <Name>Interval</Name>
+                    <Value>$Config/Interval$</Value>
+                  </Parameter>
+                  <Parameter>
+                    <Name>Format</Name>
+                    <Value>$Config/Format$</Value>
+                  </Parameter>
+                </Parameters>
+                <TimeoutSeconds>$Config/TimeoutSeconds$</TimeoutSeconds>
+              </WriteAction>
+            </MemberModules>
+            <Composition>
+              <Node ID="WA" />
+            </Composition>
+          </Composite>
+        </ModuleImplementation>
+        <InputType>System!System.BaseData</InputType>
+      </WriteActionModuleType>
+    </ModuleTypes>
+  </TypeDefinitions>
+
+  <LanguagePacks>
+    <LanguagePack ID="ENU" IsDefault="true">
+      <DisplayStrings>
+        <DisplayString ElementID="Community.DataOnDemand.WriteAction.WatchTcpConnections">
+          <Name>Data On Demand WatchTcpConnections write action</Name>
+          <Description>Displays information about TCP connections over the specified interval</Description>
+        </DisplayString>
+        <DisplayString ElementID="Community.DataOnDemand.WriteAction.WatchTcpConnections" SubElementID="Interval">
+          <Name>Period to monitor in seconds</Name>
```

Comment:
Swap this with the description and use something short like "Duration" as this won't display well in the SCOM UI otherwise.